engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -315,6 +331,17 @@ public void setEnvironment(UniverseWrapper wrapper) {
         }
     }
 
+    public int findIndex(List<WorldSetupWrapper> worldsList,String worldName) {
+        for(int i = 0 ;i<worldsList.size();i++) {
+            WorldSetupWrapper random = worldsList.get(i);
+            Name customName= random.getWorldName();
+            if(customName.toString().equals(worldName)) {
```

Comment:
`equals` or `ignoreCaseEquals` here? What happens if somebody tries to have two worlds that differ in just capitalization? What _should_ happen?

Reply:
It is not related to user.
this check the world name created by game like "perlin-0" or "flat-1" and return their indexes